ReactNative/js/screens/SearchResults/components/Results.js

```diff
@@ -8,26 +8,31 @@ import {
 import SearchUIVertical from 'browser-core-user-agent-ios/build/modules/mobile-cards-vertical/SearchUI';
 import { Provider as CliqzProvider } from 'browser-core-user-agent-ios/build/modules/mobile-cards/cliqz';
 import { Provider as ThemeProvider } from 'browser-core-user-agent-ios/build/modules/mobile-cards-vertical/withTheme';
+import themes, { baseTheme, mergeStyles } from 'browser-core-user-agent-ios/build/modules/mobile-cards-vertical/themes';
 import NativeDrawable, { normalizeUrl } from 'browser-core-user-agent-ios/build/modules/mobile-cards/components/custom/NativeDrawable';
 
 import t from '../../../services/i18n';
 
-const styles = StyleSheet.create({
+const getTheme = (theme) => mergeStyles(baseTheme, {
+
+});
+
+const getStyles = (theme) => StyleSheet.create({
   container: {
     flex: 1,
     flexDirection: 'column',
-    backgroundColor: 'gray',
+    backgroundColor: theme.backgroundColor,
   },
   footer: {
-    height: 40,
-    backgroundColor: '#464961',
+    height: 30,
+    backgroundColor: '#a2a6ae',
```

Comment:
Can you pass in these colors somehow? Otherwise, they're going to be mismatched if we change anything

Reply:
I want to have them all in sync, but I'm sure @cliqz/alexei-team will change them soon.

Reply:
That's what I mean! They're going to change, so they should only be defined in one place!